package.json

```diff
@@ -43,7 +43,7 @@
     "@material-ui/core": "^4.12.4",
     "@material-ui/icons": "^4.11.3",
     "@material-ui/lab": "4.0.0-alpha.61",
-    "@saleor/app-bridge": "^0.1.9",
+    "@saleor/app-bridge": "github:saleor/app-bridge#c09916c",
```

Comment:
Please update to the released version before the merge

Reply:
https://github.com/saleor/app-bridge/releases/tag/v0.1.10

It was released, so we can proceed